src/signature/spki.rs

```diff
@@ -472,154 +449,139 @@ mod tests {
 
     // XXX: Some of the BadDER tests should have better error codes, maybe?
 
+    // TODO need ECDSA_P256_SHA512 for these tests
     // XXX: We should have a variant of this test with a SHA-256 digest that gives
     // `VerifyWithSPKIError::UnsupportedSignatureAlgorithmForPublicKey`.
-    test_verify_signed_data!(
-        test_ecdsa_prime256v1_sha512_spki_params_null,
-        "ecdsa-prime256v1-sha512-spki-params-null.pem",
-        Err(VerifyWithSPKIError::UnsupportedSignatureAlgorithm));
-    test_verify_signed_data_signature_outer!(
-        test_ecdsa_prime256v1_sha512_unused_bits_signature,
-        "ecdsa-prime256v1-sha512-unused-bits-signature.pem",
```

Comment:
Test deleted: the signature DER is invalid, so not relevant for signature validation since we can't decode the signature itself.